docs/source/resources/cloud_integration.md

```diff
@@ -144,3 +144,59 @@ To run NVTabular on Databricks, do the following:
 
 9. Select a GPU node for the Worker and Driver.
    Once the Databricks cluster is up, NVTabular will be running inside of it.
+
+## AWS SageMaker ##
+
+[AWS SageMaker](https://aws.amazon.com/sagemaker/) is a service from AWS to "build, train and deploy machine learning" models. It automates and manages the MLOps workflow. It supports jupyter notebook instances enabling users to work directly in jupyter notebook/jupyter lab without any additional configurations. In this section, we will explain how to run NVIDIA Merlin (NVTabular) on AWS SageMaker notebook instances. We adopted the work from [Eugene](https://twitter.com/eugeneyan/) from his [twitter post](https://twitter.com/eugeneyan/status/1470916049604268035). We tested the workflow on February, 1st, 2022, but it is not integrated into our CI workflows. Future release of Merlin or Merlin's dependencies can cause issues.
+
+To run the [movielens example](https://github.com/NVIDIA-Merlin/NVTabular/tree/main/examples/getting-started-movielens) on AWS SageMaker, do the following:
+
+1. Login into your AWS console and select AWS SageMaker.
+
+2. Select `Notebook` -> `Notebook instances` -> `Create notebook instance`. Give the instance a name and select a notebook instance type with GPUs. For example, we selected `ml.p3.2xlarge`. Please review the associated costs with each instance type. As a platform identifier, select `notebook-al2-v1`. The previous platform identifier runs with TensorFlow 2.1.x and we had more issue to update it to TensorFlow 2.6.x. The `volume size` can be increased in the section `Additional configuration`.
+
+3. After the instance is running, connect to jupyter lab.
+
+4. Start a terminal to have access to the command line.
+
+5. The image contains many conda environments, which requires ~60GB of disk space. You can remove some of them to free disk space in the folder `/home/ec2-user/anaconda3/envs/`
```

Comment:
Which image contains many conda envs? Merlin containers do not contain conda env.. Can you pls clarify?

Reply:
This uses the AWS SageMaker platform. `As a platform identifier, select `notebook-al2-v1`. The previous platform identifier runs with TensorFlow 2.1.x and we had more issue to update it to TensorFlow 2.6.x.`. This defines the container, provided by AWS